db/migrations/state/0008.sql

```diff
@@ -0,0 +1,13 @@
+-- +migrate Up
+CREATE TABLE IF NOT EXISTS state.fork_id
```

Comment:
I think is good practice to define always a primary key for the tables, in this case I understand the PK is fork_id

Reply:
I don't fully understand what you mean. As you can see the fork_id is defined as Primary key
`    fork_id        BIGINT NOT NULL PRIMARY KEY,
`